sp/src/game/client/clientmode_shared.h

```diff
@@ -74,7 +74,11 @@ class ClientModeShared : public IClientMode, public CGameEventListener
 	virtual void	Disable();
 	virtual void	Layout();
 
-	virtual void	ReloadScheme( void );
+#ifdef SDK_MP
+	virtual void	ReloadScheme( bool flushLowLevel );
+#else
+	virtual void	ReloadScheme( bool flushLowLevel = false );
```

Comment:
`flushLowLevel` is non-functional in SP, but the parameter is still implemented so that code which calls the function can be one-to-one between SP and MP.